src/main/java/utility/validations/UsernameValidator.java

```diff
@@ -0,0 +1,20 @@
+package utility.validations;
+
+public class UsernameValidator {
```

Comment:
I like your approach towards username and password validation, as security by design is a key highlight of our team's project, and I'm glad that you're adopting it as well. Make sure to write test cases for different types of invalid usernames and passwords.